lib/paper_trail/model_config.rb

```diff
@@ -151,16 +145,6 @@ def cannot_record_after_destroy?
       ::ActiveRecord::Base.belongs_to_required_by_default
     end
 
-    # Some options require the presence of the `item_subtype` column. Currently
-    # only `limit`, but in the future there may be others.
-    #
-    # @api private
-    def check_presence_of_item_subtype_column(options)
-      return unless options.key?(:limit)
-      return if version_class.item_subtype_column_present?
```

Comment:
I think the `item_subtype_column_present?` method can be deleted now? That was the only call site, right?

Reply:
Correct 👍